### PR TITLE
Add compat data for input-color

### PR DESCRIPTION
**Pull request by maboa**
Testing a possible structure and format for adding compat data for input types.

See https://github.com/mdn/browser-compat-data/issues/911 and https://github.com/mdn/browser-compat-data/pull/1698

### CODE REVIEW

**Comment by maboa** [comment]
@Elchi3 @wbamberg please feel free to take a look and feedback, when you get a minute - thanks!

File: html/elements/input/color.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": false,
+                  "notes": "See <a href='https://bugzil.la/960984'>bug 960984</a> for the status of support for the <code>list</code> attribute in Firefox."
+                },
+                "firefox_android": {
+                  "version_added": "27"
```
Comment (Elchi3) [inline html/elements/input/color.json]: Like on Firefox desktop, this is `false` and the note applies here as well.

File: html/elements/input/color.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": false,
+                  "notes": "See <a href='https://bugzil.la/960989'>bug 960989</a> for the status of support for the <code>autocomplete</code> attribute in Firefox."
+                },
+                "firefox_android": {
+                  "version_added": "27"
```
Comment (Elchi3) [inline html/elements/input/color.json]: Like on Firefox desktop, this is `false` and the note applies here as well.

File: html/elements/input/color.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "color": {
```
Comment (Elchi3) [inline html/elements/input/color.json]: I think I would nest this like @wbamberg proposed here: https://github.com/mdn/browser-compat-data/pull/1698#issuecomment-390289031

So, you'd have 
- html
  - elements
    - input
      - input-color
        - __compat
          -  mdn_url: https://developer.mozilla.org/docs/Web/html/Element/input/color
          - description : "type=\"color\"
          - support: { ....
        - list
        - autocomplete

Comment (Elchi3) [reply html/elements/input/color.json]: So basically "input-color" would be better, because "color" looks like a normal attribute, which it isn't.

Comment (maboa) [reply html/elements/input/color.json]: Gotcha! That all sounds good - thanks :)

**Comment by wbamberg** [comment]
This looks great, but I think it should have a description, so it looks good when we display an aggregate table on the main [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) page:

screen shot 2018-06-06 at 4 58 53 pm

**Pull request merged**